### ISSUE TITLE
Does phase input need to be normalized with dataset statistics?

### ISSUE THREAD

**Issue by ziw-liu** [mehta-lab/microDL#221]
When deploying on a microscope system for online inference, we may not have access to dataset-level summary statistics for normalization.

During a discussion with @ieivanov and @Soorya19Pradeep, the necessity of normalizing phase input came under question. If deconvolution already guarantees zero mean for background, and the exact values of foreground pixels carry physical meaning (relative phase delay), further subtraction and scaling seems redundant for thin samples such as monolayer cell cultures. The remaining small variations can be adjusted by augmentation during model training.

@mattersoflight may have more input on whether there is empirical evidence that normalization for phase is still necessary at inference time.

**Comment by mattersoflight** [comment]
@ziw-liu , @Soorya19Pradeep , @ieivanov Great question! 

The typical dynamic range of phase (in radians) is (-0.5, 1.25)*wavelength. Interestingly, the exact dynamic range depends on two aspects of the specimen: 
* the refractive index of material (intuitive), 
* the fraction of FOV populated by dense cells/tissue (not-so-intuitive). 

Let's consider two extremes:
* If the FOV consists of uniform media, the measured mean/median phase will be zero, with zero dynamic range.
* If the FOV is 100% occupied by cells/tissue, the measured mean phase will also approach zero, with a significant dynamic range.

This difference in dynamic range does encode for confluence. For pure `phase->label` models, it does make sense to not to do any further normalization. 

The reason we decided to normalize phase (and other quantitative label-free channels) in the previous experiments was different -  we were normalizing dynamic ranges across different channels (phase, retardance, orientation, dop). These computational experiments led to [Table 2 of the paper](https://elifesciences.org/articles/55502#table2).

**Comment by ziw-liu** [comment]
> we were normalizing dynamic ranges across different channels (phase, retardance, orientation, dop)

Yes it makes sense that we scale multiple input channels to similar orders of magnitude for numerical stability. But I guess in this case the scaling factors can be fixed and not necessarily bootstrapped for each dataset?
